### ISSUE TITLE
Add CSV classifier to the AWS Glue Classifier resource

### ISSUE THREAD

**Issue by ThomasWunderlich** [terraform-providers/terraform-provider-aws#9083]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Currently the AWS Glue classifier resource supports 3/4 of the classifier types. CSV is not included for some reason (maybe it was added later?). This issue is to add the CSV classifier type to the resource 

Relevant AWS docs: https://docs.aws.amazon.com/glue/latest/dg/custom-classifier.html

### New or Affected Resource(s)

* aws_glue_classifier

### Potential Terraform Configuration

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### References

Are there any other GitHub issues (open or closed) or pull requests that should be linked here? Vendor blog posts or documentation? For example:
Related Github Issues/PRs
* #3873 

AWS Docs:
* https://docs.aws.amazon.com/glue/latest/dg/aws-glue-api-crawler-classifiers.html
* https://docs.aws.amazon.com/sdk-for-go/api/service/glue/#Glue.CreateClassifier
*  https://docs.aws.amazon.com/glue/latest/dg/custom-classifier.html

**Comment by ThomasWunderlich** [comment]
I'll note that I'm interested in working on this, potentially next week. If you don't see a related PR up by June 28 2019, then this should be considered up for grabs

**Comment by ThomasWunderlich** [comment]
I've started work on this, although it probably won't be finished by June 28th

**Comment by WheresWardy** [comment]
@ThomasWunderlich Did you manage to get anywhere with this? I might look at taking it on otherwise

**Comment by ThomasWunderlich** [comment]
@WheresWardy I actually have a partially completed branch up at https://github.com/ThomasWunderlich/terraform-provider-aws/tree/twunderlich/9083-glue-add-csv-classifier
I'm going to finish off the tests this week, if you want to take a look

**Comment by ThomasWunderlich** [comment]
The PR is currently ready and awaiting review. For those interested please thumbs up the PR so that the maintainers know how much interest there is and can prioritize it properly.  Thanks!

**Comment by bflad** [comment]
Support for a new `csv_classifier` configuration block in the `aws_glue_classifier` resource has been merged and will release with version 2.33.0 of the Terraform AWS Provider. Thanks @ThomasWunderlich for the implementation. 🎉
